microcloud/cmd/microcloud/main_init.go

```diff
@@ -179,14 +179,12 @@ func lookupPeers(s *service.Handler, autoSetup bool, iface *net.Interface, subne
 		}()
 	}
 
-	var timeAfter <-chan time.Time
+	timeoutDuration := 1 * time.Minute
```

Comment:
I don't think we need to multiply by 1 :wink:

Reply:
fixed